SuperdenseCoding/Tasks.qs

```diff
@@ -40,22 +45,19 @@ namespace Quantum.Kata.SuperdenseCoding {
     // Task 1. Entangled pair
     // Input: An array of two qubits in the |00⟩ state.
     // Goal:  Create a Bell state |Φ⁺⟩ = (|00⟩ + |11⟩) / sqrt(2) on these qubits.
-    operation CreateEntangledPair (qs : Qubit[]) : Unit {
-        // The following lines enforce the constraints on the input that you are given.
-        // You don't need to modify them. Feel free to remove them, this won't cause your code to fail.
-        EqualityFactI(Length(qs), 2, "The array should have exactly 2 qubits.");
+    operation CreateEntangledPair (q1 : Qubit, q2 : Qubit) : Unit is Adj {
 
         // ...
     }
     
     
     // Task 2. Send the message (Alice's task)
-    // Encode the message (classical bits) in the state of Alice's qubit.
+    // Encode the message (two classical bits) in the state of Alice's qubit.
     // Inputs:
     //      1) Alice's part of the entangled pair of qubits qAlice.
-    //      2) two classical bits, stored in an array.
+    //      2) two classical bits, stored as Message.
```

Comment:
This is the first time people see a type, so I'd suggest giving a quick explanation of how they can access the bits (`message::Bit1`).

Reply:
I added the first if-statement (without body) to the implementation stump below and commented that it accesses Bit1 of message. Take a look and let me know if that's what you had in mind, or if you have a better idea on how to explain named item access.